doc/metrics.md

```diff
@@ -3,24 +3,24 @@ This document aims to help users that are not familiar with metrics exposed by t
 All metrics documented here are auto-generated by the utility tool `tools/metricsdocs` and reflects exactly what is being exposed.
 
 ## Containerized Data Importer Metrics List
-### clone_progress
+### kubevirt_cdi_clone_pods_high_restart
+The number of CDI clone pods with high restart count. Type: Gauge.
+### kubevirt_cdi_clone_progress_percentage_total
```

Comment:
I'm still unsure if these metrics meet the naming conventions requested in this issue https://github.com/kubevirt/containerized-data-importer/issues/2515. I see the one for the clone percentage but not one for the remaining bytes that needs to be processed. I'd suggest to add it.

Reply:
@alromeros I agree. I think we need to change the metric to follow Prometheus suggestion.

@avlitman , Please remove this metric from this PR and keep it in the excludedMetrics list.
We will fix it as part of https://github.com/kubevirt/containerized-data-importer/issues/2515.

Reply:
+1 fixing, thanks @alromeros, @sradco

Reply:
Done